### ISSUE TITLE
Walled Templates 0.6.3 error when the Active Effect being deleted has no origin.

### ISSUE THREAD

**Issue by thatlonelybugbear** [caewok/fvtt-walled-templates#34]
```js
 Uncaught (in promise) TypeError: Cannot read properties of null (reading 'includes')  ActiveEffect.js:19
```
There are effects that do not have an origin (think DFreds CEs manually added), so the line in `ActiveEffect.js:19` will produce the above error.

Some optional chaining should be enough.

edit: though now that I think about it more, I might need to check the exact moment that this error is triggered 🤔 

edit2: yeah it is produced when you delete an DFreds CE that you have manually added from the DFreds CE app.

**Comment by caewok** [comment]
Yes, it did not occur to me that you could have an active effect without an origin!

**Comment by caewok** [comment]
Should be fixed in v0.6.4. I did not immediately see any other spots that made that assumption, so hopefully that is the only place it comes up.
